feat: Add create-llama option to RagCLI (#10405)
run-llama/llama_index: docs/use_cases/q_and_a/rag_cli.md

  -c, --chat            If flag is present, opens a chat REPL.
  -v, --verbose         Whether to print out verbose information during execution.
  --clear               Clears out all currently embedded data.

>   --create-llama        Create a LlamaIndex application with your embedded data.

Reviewer: ```suggestion
  --create-llama        Create a LlamaIndex application based on the selected files.
```

   ```
1. **Open a Chat REPL**: You can even open a chat interface within your terminal! Just run `$ llamaindex-cli rag --chat` and start asking questions about the files you've ingested.

### Create a LlamaIndex chat application

> You can also create a full-stack chat application with a FastAPI backend and NextJS frontend using your embedded data.

Reviewer: ```suggestion
You can also create a full-stack chat application with a FastAPI backend and NextJS frontend based on the files that you have selected.
```

To bootstrap the application, make sure you have NodeJS and npx installed on your machine. If not, please refer to the [LlamaIndex.TS](https://ts.llamaindex.ai/getting_started/installation) documentation for instructions.

Once you have everything set up, creating a new application is easy. Simply run the following command:

`$ llamaindex-cli rag --create-llama`

> It will call our `create-llama` package so you will need to provide several pieces of information in the to create the app, you can find more information about the `create-llama` package on [npmjs - create-llama](https://www.npmjs.com/package/create-llama#example)

Reviewer: ```suggestion
It will call our `create-llama` tool, so you will need to provide several pieces of information to create the app. You can find more information about the `create-llama` on [npmjs - create-llama](https://www.npmjs.com/package/create-llama#example)
```
